backend/tests/infrastracture/graph/test_gql_retry_backoff.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+
+from tests.conftest import mock_gql_transport_reorg_error  # noqa: F401
+
+from gql import gql
+from gql.transport.exceptions import TransportQueryError
+
+from app.extensions import gql_factory
+
+
+@pytest.fixture(autouse=True)
+def before(mock_gql_transport_reorg_error):  # noqa: F811
```

Comment:
why #noqa here and in 4. line?

Reply:
I was wandering why flake8 complains here. Turns out conftest members are autoimported :facepalm: `#noqa` removed.